### PR TITLE
jdbc

### PR DESCRIPTION
**Pull request by DadizhaO**

### CODE REVIEW

**Review by mkharkhuta — APPROVED** [review]
Good code, valid tests

File: src/test/java/Lesson15/OrderDaoImplIntegrationTest.java [hunk 1]
```diff
@@ -5,17 +5,54 @@
 
 import java.math.BigDecimal;
 import java.sql.SQLException;
+import java.util.Set;
```
Comment (SerhiiPasko) [inline src/test/java/Lesson15/OrderDaoImplIntegrationTest.java:8]: Orders and offices test must be in different classes. Also good test in our case cheack that both orders methods (getAll) return same resulr

**Review by SerhiiPasko — CHANGES_REQUESTED** [review]
Read comments

File: src/test/java/Lesson15/OrderDaoImplIntegrationTest.java (hunk 1 above)
Comment (DadizhaO) [reply src/test/java/Lesson15/OrderDaoImplIntegrationTest.java:8]: But i have this test. I compare their size. Not always sql result sorted in the same way.

**Review by SerhiiPasko — APPROVED** [review]
Good

**Pull request merged**